pkgs/applications/networking/dropbox/default.nix

```diff
@@ -103,8 +103,10 @@ in stdenv.mkDerivation {
     mkdir -p "$out/bin"
     RPATH="${ldpath}:$out/${appdir}"
     makeWrapper "$out/${appdir}/dropbox" "$out/bin/dropbox" \
-      --prefix LD_LIBRARY_PATH : "$RPATH"
+      --prefix LD_LIBRARY_PATH : "$RPATH" \
+      --suffix LD_LIBRARY_PATH : "${mesa_noglu}/lib"
   '';
+  ## FIXME: should libGL be on RPATH of qt, instead of being wrapped here?
```

Comment:
@bendlas The answer to this question is "no," because Qt loads libGL dynamically (`dlopen`). Not all features of Qt require it, and upstream wants it to work even if libGL can't be found at runtime. However, I should suffix it to Qt's default `LD_LIBRARY_PATH`.